### PR TITLE
Custom section name in user readme

### PR DESCRIPTION
**Pull request by guilyx**
Basically, allow any other section name tag than "waka" in the readme, to avoid conflicts with other projects that may use the same default one. I'll do the same PR for the other one...

Currently testing.

### CODE REVIEW

**Comment by guilyx** [comment]
@athul

**Comment by joe733** [comment]
@guilyx, thanks for the PR! It's a good idea.

But can you please re-do the PR with just one commit, since the changes are small and perfectly focused for one commit? Also add relevant notes to to the Readme's [#tweaks](https://github.com/athul/waka-readme#tweaks) section.

Thanks!

**Comment by guilyx** [comment]
Hey, actually marked it as WIP and then edited it out - testing is done atm. 
Annnnd why don't you just Squash and Merge ? Not an option ?
Will do the readme thing

**Comment by guilyx** [comment]
@joe733 Squashed all the commits in one and updated readme :)

**Review by joe733 — APPROVED** [review]
Perfect! Thanks for the PR.

> Note-to-self: Need input validation for [`os.getenv("SECTION_NAME")`](https://github.com/athul/waka-readme/pull/110/files#diff-b10564ab7d2c520cdd0243874879fb0a782862c3c902ab535faabe57d5a505e1R69-R70)

**Comment by guilyx** [comment]
Hey again @joe733 
Made it cleaner, added an input validation etc

**Comment by joe733** [comment]
Ah, thank you. But it was a `note-to-self`. Now it might requires more changes. Let me review.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -58,6 +58,7 @@
    INPUT_REPOSITORY='<REPOSITORY SLUG>'
    INPUT_COMMIT_MESSAGE='<COMMIT MESSAGE>'
    INPUT_SHOW_TITLE='True'
+   INPUT_SECTION_NAME='wakacustom'
```
Comment (joe733) [inline CONTRIBUTING.md:61]: Keep, it as `waka`

```diff
-INPUT_SECTION_NAME='wakacustom'
+INPUT_SECTION_NAME='waka'
```

File: main.py [hunk 2]
```diff
@@ -336,7 +342,7 @@ def churn(old_readme: str, /) -> str | None:
     if not (waka_stats := fetch_stats()):
         logger.error('Unable to fetch data, please rerun workflow')
         sys.exit(1)
-
+
```
Comment (joe733) [inline main.py:345]: Stray (tab) white space

**Review by joe733 — APPROVED** [review]

**Comment by joe733** [comment]
I do want to make changes, but for now let it be.

**Pull request merged**